src/pyproject_hooks/__init__.py

```diff
@@ -20,7 +24,5 @@
     "default_subprocess_runner",
     "quiet_subprocess_runner",
     "BuildBackendHookCaller",
+    "SubprocessRunner",
```

Comment:
I don't think this is compatible with the `if TYPE_CHECKING`  condition - it looks like doing `from pyproject_hooks import *` would fail with an AttributeError when it tries to get `SubprocessRunner`.

Does mypy accept anything like:

```python
if TYPE_CHECKING:
    __all__.append("SubprocessRunner")
```

Reply:
Ah, good catch. `__all__ += [...]` appears to work.

Reply:
Honestly forgot `__all__` has an impact on the runtime, lol.

Reply:
:laughing: Thanks!